src/Particle/Validator/Rule/Between.php

```diff
@@ -28,7 +33,8 @@ class Between extends Rule
      * @var array
      */
     protected $messageTemplates = [
-        self::NOT_BETWEEN => '{{ name }} must be between {{ min }} and {{ max }}'
+        self::TOO_BIG => '{{ name }} is too big, upper limit is {{ max }}',
```

Comment:
the reason I did not want to have less than , means it cannot have the "inclusive" semantically...